qiskit/algorithms/minimum_eigensolvers/vqe.py

```diff
@@ -0,0 +1,333 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The variational quantum eigensolver algorithm."""
+
+from __future__ import annotations
+
+import logging
+from time import time
+from collections.abc import Callable, Sequence
+
+import numpy as np
+
+from qiskit.algorithms.gradients import BaseEstimatorGradient
+from qiskit.circuit import QuantumCircuit
+from qiskit.opflow import PauliSumOp
+from qiskit.primitives import BaseEstimator
+from qiskit.quantum_info.operators.base_operator import BaseOperator
+
+from ..exceptions import AlgorithmError
+from ..list_or_dict import ListOrDict
+from ..optimizers import Optimizer, Minimizer, OptimizerResult
+from ..variational_algorithm import VariationalAlgorithm, VariationalResult
+from .minimum_eigensolver import MinimumEigensolver, MinimumEigensolverResult
+from ..observables_evaluator import estimate_observables
+from ..utils import validate_initial_point, validate_bounds
+
+logger = logging.getLogger(__name__)
+
+
+class VQE(VariationalAlgorithm, MinimumEigensolver):
+    r"""The variational quantum eigensolver (VQE) algorithm.
+
+    VQE is a hybrid quantum-classical algorithm that uses a variational technique to find the
+    minimum eigenvalue of a given Hamiltonian operator :math:`H`.
+
+    The VQE algorithm is executed using an :attr:`estimator` primitive, which must be specified as
+    the first argument on instantiation.
+
+    An instance of ``VQE`` also requires an :attr:`ansatz` parameterized :class:`.QuantumCircuit` to
+    prepare the trial state :math:`|\psi(\vec\theta)\rangle`, as well as a classical
+    :attr:`optimizer`. The optimizer varies the circuit parameters :math:`\vec\theta` such that the
+    expectation value of the operator on the corresponding state approaches a minimum,
```

Comment:
```suggestion
    expectation value of the operator on the corresponding state approaches a minimum.
```

Reply:
@woodsp-ibm I accepted this change, but actually, looking again at the context, the reason I ended this line with a comma is that I end the sentence after the equation on the following line, which is what I'd do in a paper, but perhaps isn't the standard here.

Reply:
I've reverted this change, but I can unrevert it and instead remove the "." from the equation below :)

Reply:
Ah ok, I guess I did not really notice that. Had it said something like `approaches a minimum as defined by the following equation:`  it might be more self-evident. Maybe its easier to see when looking at the resultant html. It just felt/seemed like the end of the sentence to me I must admit!

Reply:
(I also forgot the context when I looked at your suggestion.) At the last full build it looked like this
<img width="746" alt="Screenshot 2022-09-26 at 21 22 06" src="https://user-images.githubusercontent.com/14238050/192372704-6a64cf64-c74a-4c23-821a-b5ab021cead6.png">
I think it's clear when rendered, but happy to change it.

Reply:
Lets stick with what you have then.